CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Unreleased
+
+- Remove unused batch consumer
```

Comment:
Should we add a "Breaking: " to this as a clue that we need to bump the major version.

Might want to drop the word unused here as we, the suppliers of the gem, just think it's not needed. We create a bit of a paradox if we describe it as unusued but remove it with a major change.

Reply:
Yup all makes sense, have updated